### PR TITLE
Feature/leecalcote/eks

### PR DESCRIPTION
**Pull request by venkatasankarapu**
edit  of eks.md file

### CODE REVIEW

File: docs/pages/installation/generate_kubeconfig_eks.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+get_secret_name_from_service_account() {
+    echo -e "\\nGetting secret of service account ${SERVICE_ACCOUNT_NAME} on ${NAMESPACE}"
+    SECRET_NAME=$(kubectl get sa "${SERVICE_ACCOUNT_NAME}" --namespace="${NAMESPACE}" -o json | jq -r .secrets[].name)
+    echo "Secret name: ${SECRET_NAME}"
+}
+
+extract_ca_crt_from_secret() {
+    echo -e -n "\\nExtracting ca.crt from secret..."
+    kubectl get secret --namespace "${NAMESPACE}" "${SECRET_NAME}" -o json | jq \
+    -r '.data["ca.crt"]' | base64 -D > "${TARGET_FOLDER}/ca.crt"
```
Comment (leecalcote) [inline docs/pages/installation/generate_kubeconfig_eks.sh]: On what type of OSes is the `-D` compatible with?

File: docs/pages/installation/generate_kubeconfig_eks.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```
Comment (leecalcote) [inline docs/pages/installation/generate_kubeconfig_eks.sh]: Is there anything EKS-specific in here or is this the same script verbatim for GKE?

Comment (venkatasankarapu) [reply docs/pages/installation/generate_kubeconfig_eks.sh]: I suppose to change to -d, but I didn't change. will change it again and do PR.

File: docs/pages/installation/generate_kubeconfig_eks.sh (hunk 1 above)
Comment (venkatasankarapu) [reply docs/pages/installation/generate_kubeconfig_eks.sh]: I am not sure, Will check on that and get back to you lee.

**Pull request merged**